scripts/src/test/dummyfiles/DummyFile1.kt

```diff
@@ -0,0 +1 @@
+import android.support.v7.app
```

Comment:
Add something valid file

Reply:
Also add eof

Reply:
Done regarding eof.
Regarding what content we should add here, I am not sure.
@BenHenning your thoughts?